app/battle_preparations/inventory.py

```diff
@@ -0,0 +1,21 @@
+class Inventory:
```

Comment:
instead of class Inventory create class Knight and save all of characteristics there as attributes